release-1.27: Add docs for enabling Istio native nftables feature (#1122)
istio-ecosystem/sail-operator: docs/common/istio-nftables.md

```sh
kubectl create ns test-ns
kubectl label namespace test-ns istio-injection=enabled
kubectl apply -n test-ns -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/curl/curl.yaml
```

Attach a debug container and you will get the following rules in the `istio-proxy` container:

```sh
kubectl -n test-ns debug --image istio/base --profile netadmin --attach -t -i \
  "$(kubectl -n test-ns get pod -l app=curl -o jsonpath='{.items..metadata.name}')"
root@curl-6c88b89ddf-kbzn6:$ nft list ruleset

```

Verify the connectivity between two pods is working. For example, deploy a httpbin application using the following step:

```sh
kubectl apply -n test-ns -f samples/httpbin/httpbin.yaml
kubectl exec -n test-ns "$(kubectl get pod -l app=curl -n test-ns -o jsonpath={.items..metadata.name})" -c curl -n test-ns -- curl http://httpbin.test-ns:8000/ip -s -o /dev/null -w "%{http_code}\n"

200
```

More guidelines: [Debugging Guidelines](https://github.com/istio/istio/tree/master/tools/istio-nftables/pkg#debugging-guidelines)

### Upgrade

> The migration of using the existing Istio iptables backend to nftables backend can be done by upgrading Istio. The following example installs an Istio control plane with the iptables backend and a sample application curl in a data plane namespace test-ns.

Reviewer: ```suggestion
The migration from iptables backend to nftables backend can be done by upgrading Istio. The following example installs an Istio control plane with the iptables backend and a sample application in test-ns namespace.
```

```sh

> istioctl install -y

Reviewer: As this is a Sail Operator doc, lets use the installation mechanisms supported in Sail Operator.

> [Return to Project Root](../README.md)

Reviewer 2: Hey, can you please convert these files to ASCII doc? We now have all the docs under the docs folder in ASCII format

Author: yes, I pushed another commit about adding a ascii one. I'm checking. If the conversion looks correct , I will remove the markdown file in another commit after review.
